drivers/input/Kconfig

```diff
@@ -65,7 +65,7 @@ endif # FF_AW86225
 endif # INPUT_FF
 
 config INPUT_MOUSE
-	bool
+	bool "Enable mouse support"
```

Comment:
@xiaoxiang781216 what do you think?

Reply:
In the rpmsg device scenario, the actual input device is located on the remote core, and there is no real hardware driver on local side. In this case, we need to be able to enable the input device option separately so that the application layer can seamlessly use the remote input device.
CC @HongChao6 @xiaoxiang781216

Reply:
or can we remove #ifdef/#else around the public struct, macro and function from .h?

Reply:
@terry0012 and @HongChao6 I understand that, but after you let user to see and enable these symbols it is expected to work. So if user enable Mouse or Keyboard they will expect it so work, but it will not, because it depends on some external driver. So, I think it should to become a new session, like Sensors Driver, after you enable it you should be presented with all Sensors drivers. It means all "select" now should become "depends on"